src/Faithlife.Build/DotNetClassicTool.cs

```diff
@@ -62,11 +62,14 @@ public sealed class DotNetClassicTool
 		var packagesPath = Environment.GetEnvironmentVariable("NUGET_PACKAGES") ??
 			Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".nuget", "packages");
 
-		// Find a matching package. If using a specific version, there will be exactly one match.
-		// If using floating versions, there may be multiple matches, so take the latest semantic version.
-		var packagePath = Directory.GetDirectories(Path.Combine(packagesPath, packageName.ToLowerInvariant()), packageVersion)
-			.OrderByDescending(fullPath => NuGetVersion.Parse(Path.GetFileName(fullPath)))
-			.FirstOrDefault();
+		var packageDirectories = Directory.GetDirectories(Path.Combine(packagesPath, packageName.ToLowerInvariant()))
+			.Select(fullPath => NuGetVersion.Parse(Path.GetFileName(fullPath)));
```

Comment:
This will enumerate the directories twice without a `.ToList()`.

Also, I'd feel a little more comfortable if we used `TryParse` instead of `Parse`, but it's unlikely to be a problem.